pkg/cluster/admin_actions.go

```diff
@@ -0,0 +1,19 @@
+package cluster
+
+import (
+	"context"
+
+	"github.com/openshift/openshift-azure/pkg/api"
+)
+
+func (u *simpleUpgrader) ReimageAndRestart(ctx context.Context, cs *api.OpenShiftManagedCluster, vmScalesetName string, instanceId string) *api.PluginError {
+	err := u.vmc.Reimage(ctx, cs.Properties.AzProfile.ResourceGroup, vmScalesetName, instanceId, nil)
+	if err != nil {
+		return &api.PluginError{Err: err, Step: api.PluginStepReimageAndRestartVmReimageStep}
+	}
+	err = u.vmc.Restart(ctx, cs.Properties.AzProfile.ResourceGroup, vmScalesetName, instanceId)
```

Comment:
Is the restart superfluous?  I think Reimage on a running machine also restarts it?

Reply:
I think it is, the docs https://docs.microsoft.com/en-us/rest/api/compute/virtualmachinescalesetvms/reimage say it upgrades the OS, so in theory after reimage there is a "first boot"

Reply:
i tried this out in the portal and reimaging a running vm does include a restart. I'll change the entire API from `ReimageAndRestart` to `Reimage`

![image](https://user-images.githubusercontent.com/5978370/53294843-3c4e8100-37ef-11e9-9a85-d006f232fbe6.png)